uaclient/cli.py

```diff
@@ -1271,7 +1271,7 @@ def action_disable(args, *, cfg, **kwargs):
 @assert_root
 @assert_attached(_create_enable_disable_unattached_msg)
 @assert_lock_file("pro enable")
-def action_enable(args, *, cfg, **kwargs):
```

Comment:
Why do we need the `extra_args` attribute ? Can't we use `kwargs` here ?

Reply:
Updated